content/departments/product-engineering/engineering/code-graph/code-intelligence/glossary.md

```diff
@@ -0,0 +1,374 @@
+# Code Intel Glossary
```

Comment:
So I hate to say this, but ... can we not use this term anymore? Our umbrella term was officially stolen in this week's company meeting.

Reply:
We have "Language Tools and Language Platform and Code Navigation". Rolls off the tongue.